examples/webgl_loader_xfile.html

```diff
@@ -0,0 +1,307 @@
+                                dashing.setLoop(THREE.LoopPingPong);
+                                // dashing.play();
+                                actions['dashing'] = dashing;
+
+                                var damage = Models[i].mixer.clipAction('damage');
+                                damage.setLoop(THREE.LoopRepeat);
+                                //damage.play();
+                                actions['damage'] = damage;
+
+                                var ActionKeys = Object.keys(actions);
+                                var dmy = {};
+                                dmy.gui = "";
+                                dmy.action = "";
+                                gui.add(dmy, 'gui');
+                                gui.add(dmy, 'action', ActionKeys).onChange(function (v) {
+                                    animates[0].stopAllAction();
+                                    actions[v].play();
+                                });
+                            }
+                            ///////////
+                        }
+                    }
+                }
+
+                object = null;
+            }, onProgress, onError);
+
+
+            renderer = new THREE.WebGLRenderer();
+            renderer.setPixelRatio(window.devicePixelRatio);
+            renderer.setSize(window.innerWidth, window.innerHeight);
+            renderer.setClearColor(0x666666);
+            container.appendChild(renderer.domElement);
+
+            controls = new THREE.OrbitControls(camera, renderer.domElement);
+            controls.target.set(0, 12, 0);
+            camera.position.set(2, 18, 28);
+            camera.up.set(0, 1, 0);
+
+            var light = new THREE.DirectionalLight(0xeeeeff, 2);
+            light.position.set(10, 100, 1).normalize();
+            scene.add(light);
+
+            light = new THREE.DirectionalLight(0xaa5555);
+            light.position.set(-1, -1, -1).normalize();
+            scene.add(light);
+
+            controls.update();
+            window.addEventListener('resize', onWindowResize, false);
+            animate();
+
+        }
+        function onWindowResize() {
+            camera.aspect = window.innerWidth / window.innerHeight;
+            camera.updateProjectionMatrix();
+            renderer.setSize(window.innerWidth, window.innerHeight);
+        }
+        //
+        function animate() {
+
+            requestAnimationFrame(animate);
+            var nowTime = Date.now();
+            var dulTime = nowTime - LastDateTime;
+            LastDateTime = nowTime;
+
+            if (animates != null && animates.length > 0) {
+                for (var i = 0; i < animates.length; i++) {
+                    animates[i].update(dulTime);
+                }
+            }
+
+            if (Models != null && Models.length > 0) {
+                if (skeletonHelper != null) { skeletonHelper.update(); }
```

Comment:
It can be done in one `if` statement, since there is only one block inside.
Ex: 
```javascript
if (Models != null && Models.length > 0 && skeletonHelper != null) {
 // ...
}
```